package.json

```diff
@@ -63,7 +63,8 @@
     "lint-staged": "^10.2.8",
     "prettier": "^2.0.5",
     "react-test-renderer": "^16.13.1",
-    "start-server-and-test": "^1.11.0"
+    "start-server-and-test": "^1.11.0",
+    "zipcodes": "^8.0.0"
```

Comment:
Since this is for the UI it will need to not be a devDependency

Reply:
Ah, blame me for that :)

Reply:
No problem, I'll get on that right now!

Reply:
Ok, done.